docs/deployment/helm.md

```diff
@@ -65,6 +65,67 @@ export NFD_NS=node-feature-discovery
 helm install nfd/node-feature-discovery --set nameOverride=NFDinstance --set master.replicaCount=2 --namespace $NFD_NS --create-namespace
 ```
 
+## Upgrading the chart
+
+To upgrade the `node-feature-discovery` deployment to {{ site.release }} via Helm.
+
+### From v0.7 and older
+
+Please see the [uninstallation guide](https://kubernetes-sigs.github.io/node-feature-discovery/v0.7/get-started/deployment-and-usage.html#uninstallation).
+And then follow the standard [deployment instructions](#deployment).
+
+### From v0.8 - v0.11
+
+Helm deployment of NFD was introduced in v0.8.0.
+
+```bash
+export NFD_NS=node-feature-discovery
+# Uninstall the old NFD deployment
+helm uninstall node-feature-discovery --namespace $NFD_NS
+# Update Helm repository
+helm repo update
+# Install the new NFD deployment
+helm upgrade --install node-feature-discovery nfd/node-feature-discovery --namespace $NFD_NS --set master.enable=false
+# Wait for NFD Worker to be ready
```

Comment:
what will happen if we don't wait here ?

Reply:
If master goes live before workers (for some unexpected reason) master won't find NodeFeatures for the nodes and could trigger a label deletion event

Reply:
ack